### ISSUE TITLE
placeholders and empty states

### ISSUE THREAD

**Issue by oliver-sanders** [cylc/cylc-ui#332]
Stand-in content for empty / null states to make users aware of the state (rather than just displaying a blank page).

Modern UI design seems to have settled on a greyscale background image for empty states these days. Clear, concise and simple to implement.

I believe material design has built-in support.

Some examples of places where we need empty state placeholders and some ideas as to what they could look like:

![placeholders](https://user-images.githubusercontent.com/16705946/70871159-063df000-2001-11ea-8e34-2923051239d9.png)

Missing placeholders:

* [ ] Workflow: workflow stopped
* [ ] Workflow: connecting/loading (spinner / foreground icon)
* [ ] Workflow: network error (unable to get data)
* [ ] Workflow: no active view
* [ ] GScan: no workflow
* [ ] GScan: connecting/loading

**Pull requests welcome!**

**Comment by MetRonnie** [comment]
Vuetify has a [`VEmptyState` component](https://vuetifyjs.com/en/components/empty-states/#usage) that should make this easy

**Comment by oliver-sanders** [comment]
I'm not quite sure what `VEmptyState` brings to the party, it's trivial to roll your own:

```
<div v-if="workflow.status === 'stopped'">
  <!-- placeholder -->
</div>
<div v-else>
   <!-- content -->
</div>
```

**Comment by MetRonnie** [comment]
Yes but even trivialier using that component. No need to roll your own styles
